lib/erlef_web/templates/blog/index.html.eex

```diff
@@ -0,0 +1,71 @@
+<div class="container mt-2">
+  <% post1 = Enum.at(@posts, 0) %>
+  <%= unless is_nil(post1) do %> 
+    <div class="jumbotron p-4 p-md-5 text-white rounded bg-dark">
+      <div class="col-md-6 px-0">
+        <h1 class="display-4 font-italic"><%= post1.title %></h1>
+        <p class="lead my-3"><%= post1.excerpt %></p>
+        <p class="lead mb-0">
+        <%= link("Continuing Reading", to: Routes.blog_path(@conn, :show, @topic, post1.slug), class: "stretched") %>
+      </div>
+    </div>
+  <% end %>
+
+  <% latest = Enum.take(@posts, 3) %>
+  <%= if length(latest) == 3 do %>
+    <% [_, post2, post3] = latest %>
```

Comment:
Noting that if there's not at least 3 posts, we don't bother with the second row.